### DISCUSSION TITLE
PUA:Win32/Sepdot malware detected in Windows build?

### DISCUSSION THREAD

**Discussion by MartinCapousek** [helix-editor/helix#4197] [Q&A]
Microsoft Defender reporting mentioned malware. Do you know why? False positive?

**Comment by dead10ck** [comment]
I can tell you pretty confidently that there is no malware in the source code.

**Comment by MartinCapousek** [comment]
I wouldn't expect that for a widely used open-source project :). However, the last two Windows releases didn't pass (at least for me) checks by Microsoft Defender. The author of the build does not have to be the author of the source code itself... that is why I'm asking. As I'm not familiar with Rust (not able to build project by myself) I tried to use provided builds.

**Reply by dead10ck** [reply]
If you're downloading the builds from the GitHub Releases, then you can be confident MS Defender is incorrect.
